### PR TITLE
OCM-8285 | test: implement no-cni(cilium) supporting

### PR DESCRIPTION
**Pull request by yingzhanredhat**
log:
https://privatebin.corp.redhat.com/?0b978894d249f696#9bzbGrGVpgxtorSWfUetCKFCFtnHJRaVcxwPHmLZ8QuA

### CODE REVIEW

File: tests/utils/common/helper.go [hunk 1]
```diff
@@ -8,3 +8,45 @@ func ReadENVWithDefaultValue(envName string, fallback string) string {
...
+func StartsWith(st string, substring string) (flag bool) {
+	if len(st) < len(substring) {
+		return
+	}
+	flag = (st[:len(substring)] == substring)
+	return
+}
+
+// EndsWith will return the bool value that whether the st is end with substring
+func EndsWith(st string, substring string) (flag bool) {
```
Comment (xueli181114) [inline tests/utils/common/helper.go:45]: Can we use `strings.HasSuffix()`?

File: tests/utils/common/helper.go [hunk 2]
```diff
@@ -8,3 +8,45 @@ func ReadENVWithDefaultValue(envName string, fallback string) string {
...
+// Rstrip will return the string right striped with substring
+func Rstrip(st string, substring string) string {
+	if EndsWith(st, substring) {
+		st = st[:len(st)-len(substring)]
+	}
+	return st
+}
+
+// StartsWith return bool whether st start with substring
+func StartsWith(st string, substring string) (flag bool) {
```
Comment (xueli181114) [inline tests/utils/common/helper.go:36]: Let's use `strings.HasPrefix()`

File: tests/utils/config/support.go [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+	if err != nil {
+		log.Logger.Errorf("%s", stdout)
+		return err
+	}
+
+	return err
+}
+
+func GetKubeconfigDummyFunc() {
+	// TODO: create IDP to get kubeconfig
```
Comment (radtriste) [inline tests/utils/config/support.go:77]: Are you planning on implementing that part in this PR ? 
Or maybe reference an issue for follow-up ?

**Pull request merged**